centreon/www/front_src/src/Resources/translatedLabels.ts

```diff
@@ -287,3 +287,8 @@ export const labelForcedCheckDescription =
 export const labelForcedCheckCommandSent =
   'Forced check command sent ! Please refresh the listing to update the data.';
 export const labelUnit = 'Unit';
+export const labelNeedHelpWithSearchBarUsage =
+  'Need help with search bar usage ?';
+export const labelFindExplanationsAndExamples =
+  'Find explanations and examples';
+export const labelHere = 'here';
```

Comment:
is there a French translation for "here"?

Reply:
Yes, it already exists.